docs/dev/setup/bamboo-bitbucket-jira.rst

```diff
@@ -59,7 +59,8 @@ Configure Bamboo, Bitbucket and Jira
 
 By default, the Jira instance is reachable under ``localhost:8081``, the
 Bamboo instance under ``localhost:8085`` and the Bitbucket instance
-under ``localhost:7990``.
+under ``localhost:7990``. You will probably have to change the respective URLs in
+``application-local.yml`` (e.g. set ``artemis.version-control.url`` to ``http://localhost:7990``).
```

Comment:
As fast as I can see, they already have that name in application-artemis.yml and in the example in the server setup for the application-local.yml.
Only this location here uses the version without the `http://`

Reply:
The default URLs in `application-artemis.yml` [are the ones from TUM](https://github.com/ls1intum/Artemis/blob/develop/src/main/resources/config/application-artemis.yml#L20). 

I will move this info to the part of the guide where admin username + password for Bitbucket/Jira/Bamboo are set, as the URLs are also changed there. When I followed the instructions, I think I didn't notice that and just changed the credentials in my local configuration.